plugins/inputs/kube_inventory/kube_state.go

```diff
@@ -142,6 +146,20 @@ func atoi(s string) int64 {
 	return int64(i)
 }
 
+func convertQuantity(s string) float64 {
```

Comment:
We want to make the conversion to int64

> Note that the quantity will NEVER be internally represented by a
> floating point number. That is the whole point of this exercise.
>
> This format is intended to make it difficult to use these numbers without
> writing some sort of special handling code in the hopes that that will
> cause implementors to also use a fixed point implementation.

It seems clear that the bytes values should be in bytes, but we may have to alter the units for cpu_units.  Is there a definition anywhere of what exactly a cpu_unit is?

Reply:
yep, that's why they're doing the whole string representation in the first place, so they don't say a machine has 1/10th a cpu, but instead, 100 milicores. we need it a float for cpu cores, bytes maybe not so if its a big deal i'll convert that to int
https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu

Reply:
I'm not against reporting it in millicpu either, if that's the smallest unit, the field would need to be named `blah_millicpu`.  @fntlnz What do you think?